### DISCUSSION TITLE
Flood vulnerabillity for bridges

### DISCUSSION THREAD

**Discussion by hkayabilisim** [TomorrowsCities/tomorrowscities#4] [General]
To implement flood damage assessment in transportation networks, flood vulnerability curves for bridges are needed. Do you @fabrizio-nocera  have sample curves and preferably a pseudo-code related to this.

**Comment by fabrizio-nocera** [comment]
@hkayabilisim, for the analyses on the transportation network subject to flood, we do not need flood vulnerability curves for bridges. We assume them not to be vulnerable to flood. The impact is quantified in terms of reduction of accessibility/inaccessibility to places of interest, such as hospitals because the roads are flooded when the water height is above a threshold value, making the road segment impassable.

**Comment by hkayabilisim** [comment]
Thank you @fabrizio-nocera! Ok no curves for bridges then. What about the threshold for water height. Do you suggest a specific default value? And the secondly, do you think a user should be able to change it very easily or it should be managed only by an admin user?
